api/src/test/java/org/pmiops/workbench/institution/InstitutionServiceTest.java

```diff
@@ -20,71 +27,89 @@
 
 @RunWith(SpringRunner.class)
 @DataJpaTest
-@Import({InstitutionServiceImpl.class})
+@Import({InstitutionServiceImpl.class, InstitutionMapperImpl.class})
 public class InstitutionServiceTest {
   @Autowired private InstitutionService service;
+  @Autowired private UserDao userDao;
+  @Autowired private VerifiedInstitutionalAffiliation verifiedInstitutionalAffiliation;
 
-  private final Institution testInst =
+  private final Institution TEST_INST =
       new Institution().shortName("test").displayName("this is a test");
 
-  // the DB converts nulls to empty lists
-  private final Institution testInstAfterRT =
+  // the mapper converts nulls to empty sets
+  private final Institution TEST_INST_AFTER_RT =
```

Comment:
[super nit] personally I'd expand RT to ROUNDTRIP or ROUND_TRIP to save someone the effort of parsing out what this means.

Reply:
done